platform-dom/src/main/java/com/softicar/platform/dom/elements/testing/node/tester/AbstractDomNodeTester.java

```diff
@@ -86,27 +88,34 @@ public String getInputValue() {
 
 	// ------------------------------ events ------------------------------ //
 
-	public void sendEvent(DomEventType type) {
+	public AbstractDomNodeTester<N> sendEvent(DomEventType type) {
 
+		Assert.assertFalse("Trying to send %s event to disabled node.".formatted(type), isNodeDisabled());
```

Comment:
added this assertion because it cost me about 1/2h to come to the conclusion that the fine button was disabled all the time, because of missing user permissions ;)

Reply:
btw: By our current definitions, this is an API break... by exclusion, because it's neither a new feature nor a bugfix.

It's not a bugfix because, after all, it might be perfectly fine to click a disabled element in a test -- to ensure that _nothing_ happens when an actual user clicks the element. This is a deliberate change of intended behavior.

Reply:
not sure about this, but we will increase the version anyway, i assume